.github/workflows/ci.yml

```
@@ -35,6 +35,17 @@ jobs:
     steps:
     - uses: actions/checkout@v4
 
+    - name: Clone the Shuttle main repository
+      if: ${{ github.ref == 'refs/heads/develop' }}
+      run: git clone https://github.com/shuttle-hq/shuttle.git
```

Make the clone a bit faster
```suggestion
      run: git clone --depth 1 https://github.com/shuttle-hq/shuttle.git
```